frontend/src/metabase/query_builder/components/view/sidebars/QuestionDetailsSidebar.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import SidebarContent from "metabase/query_builder/components/SidebarContent";
+
+function QuestionDetailsSidebar() {
+  return (
+    <SidebarContent className="full-height px1">
+      question details sidebar
+    </SidebarContent>
+  );
+}
```

Comment:
Not sure why it contains just "question details sidebar"? Or is it a WIP?

Reply:
ya, this is a WIP, that's just random text for no real reason at all. lots of changes and I didn't want a huge PR--my next 10-20 PRs in this feature branch will handle filling it out to look like this:

![Screen Shot 2021-04-23 at 8 33 11 AM](https://user-images.githubusercontent.com/13057258/115894853-99194680-a40e-11eb-8641-bdb2f777c674.png)